books/clients/livelibru.py

```diff
@@ -0,0 +1,89 @@
+import requests
```

Comment:
Let's add module-level comment explaining what it does

https://google.github.io/styleguide/pyguide.html#382-modules